lib/driver.js

```diff
@@ -264,11 +264,18 @@ class AndroidUiautomator2Driver extends BaseDriver {
     // ADB instance
     this.adb = await androidHelpers.createADB(this.opts);
 
-    if (await this.adb.getApiLevel() < 21) {
+    const apiLevel = await this.adb.getApiLevel();
+
+    if (apiLevel < 21) {
       logger.errorAndThrow('UIAutomation2 is only supported since Android 5.0 (Lollipop). ' +
         'You could still use other supported backends in order to automate older Android versions.');
     }
 
+    if (apiLevel >= 28) { // Android P
+      logger.warn('Relaxing hidden api policy');
```

Comment:
Can we change this to `logger.info`? Two reasons: 

1. It matches the log level further down where you reset the API Policy to the default: 

    ```js
    if (await this.adb.getApiLevel() >= 28) { // Android P
      logger.info('Restoring hidden api policy to the device default configuration');
      await this.adb.setDefaultHiddenApiPolicy();
    }
    ```

2. It's not really a warning if it's always set and not something we can actually fix/remediate.

Plus, it shows up in our console logs, which are warning only and it's noisy: 

![image](https://user-images.githubusercontent.com/180819/82927767-6cc72280-9f3e-11ea-9e28-6e72665e5489.png)

Reply:
@joshuapinter Feel free to push a PR

Reply:
Excellent, that's what I wanted to hear.

![source](https://user-images.githubusercontent.com/180819/82939439-9be68f80-9f50-11ea-87bc-9b82a9a8fffb.gif)

Reply:
PR pushed at #396